custom/icds_reports/utils/aggregation_helpers/distributed/agg_awc_daily.py

```diff
@@ -123,6 +120,27 @@ def update_query(self):
             'date': self.date
         }
 
+    def update_launched_query(self):
+        return """
+        UPDATE "{tablename}" agg_awc SET
+           num_launched_states = CASE WHEN ut.all_cases_household>0 THEN 1 ELSE 0 END,
+           num_launched_districts = CASE WHEN ut.all_cases_household>0 THEN 1 ELSE 0 END,
+           num_launched_blocks = CASE WHEN ut.all_cases_household>0 THEN 1 ELSE 0 END,
+           num_launched_supervisors = CASE WHEN ut.all_cases_household>0 THEN 1 ELSE 0 END,
+           num_launched_awcs = CASE WHEN ut.all_cases_household>0 THEN 1 ELSE 0 END
+        FROM ( SELECT
```

Comment:
I think it can't be done directly updating the table like this. This would need to store this result in a temp table and then use that temp table as the underlying UCR is a distributed table.

Reply:
Made changes accordingly, please review.